### DISCUSSION TITLE
Implementing Widget (possibly Rainmeter skin)

### DISCUSSION THREAD

**Discussion by IvankovPaul** [timschneeb/GalaxyBudsClient#76] [General]
Hello ! I ask you to make a widget for Windows on the desktop. In order to enable the blocking of the buttons or background noise, volume. Perhaps this can be implemented via rainmeter...)

**Comment by ArthurWolfhound** [comment]
Идея здесь останется это уж точно, но она точно не попадет в приоритетный бэклог - разработчик сейчас переписывает приложение с нуля для того чтобы добавить кроссплатформенность и вообще улучшить прогу. А с такой переписью могут вылезти новые баги и все остальное... 

Но спасибо за идею

**Comment by IvankovPaul** [comment]
Жаль( 
Была бы очень полезная плюшка)

**Reply by ArthurWolfhound** [reply]
![image](https://user-images.githubusercontent.com/16789106/102063629-f36e1f00-3dfe-11eb-9b50-d13bb076d855.png)
По крайней мере вот так можно, пока что - все что есть...

**Comment by ArthurWolfhound** [comment]
Вроде бы как теперь можно назначать клавиатурные комбинации, но я ещё сам не тестировал.
